docs/index.rst

```diff
@@ -97,6 +97,22 @@ persisted, and so won't survive after the container is finished.
     via the `Dockerfile`. Commits to master will result in the Dockerfile being
     rebuilt on the docker hub.
 
+Hacking on the Docker image
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to make changes to the Olympia Docker image, use the
```

Comment:
Maybe change "If you want to make changes to the Olympia Docker image," to "If you want to test out changes to the Olympia Docker image locally,"